subcommands/keys/tuf_updates_init.go

```diff
@@ -17,14 +19,28 @@ func init() {
 	}
 	initCmd.Flags().StringP("changelog", "m", "", "Reason for doing this operation. Saved in root metadata to track change history.")
 	_ = initCmd.MarkFlagRequired("changelog")
+	initCmd.Flags().BoolP("first-time", "", false, "Used for the first customer rotation. The command will download the initial root key.")
+	initCmd.Flags().StringP("keys", "k", "", "Path to <offline-creds.tgz> used to store initial root key.")
```

Comment:
i was just playing with this. I started wondering "why does this command need keys to initiate". Then I realized it's only when `--first-time` is used. Maybe you collapse these two options into one, `--first-time <path to offlinecreds.tgz>`

Reply:
We use the `-k` argument in various waves and keys subcommands.
So, I think it makes sense to keep it here the same just to make the user feel safe; even at the cost of extra 3 characters.

Reply:
i like your thinking about making the user feel safe. I guess its subjective - this flag made me feel *unsafe* because I didn't see why i need root keys to do the `init` command. But - if you read closely it becomes more clear. This is "danger zone" commands, so I guess this is fine and to be expected.